### DISCUSSION TITLE
Announcing GCM's new roadmap!

### DISCUSSION THREAD

**Discussion by ldennington** [git-ecosystem/git-credential-manager#1203] [General]
Hello GCM community! 👋

We are excited to announce that we’ve begun maintaining a [roadmap](https://github.com/git-ecosystem/git-credential-manager/milestones) of planned GCM features. 💡**Pro tip:** When viewing the roadmap, choose **Furthest due date** from the **Sort** menu in the upper-right corner to view the milestones in ascending order.

We hope this will help bring clarity on the planned efforts of the project and their timelines to you all. There are a few things to note:

1. In an effort to stay aligned with the Git project, we’ve created milestones that coincide with Git’s upcoming releases. Our goal is to consistently publish concrete plans for the next two releases, with a new milestone created each time a release completes.
0. Items that we’re tracking but haven’t yet been assigned to a release will fall into the **Future** milestone.
0. This roadmap is meant to track major projects. While they are not tracked on the roadmap, project maintainers will continue to address/allocate time for bugs and questions as part of their weekly review meetings.

Many thanks to @hickford for recommending this approach!

Questions? Comment below! We’re all ears 👂.

**Comment by oxSTERBENxo** [comment]
❤️

**Reply by Eliyanbeyan** [reply]
Hello

**Comment by spongemike2** [comment]
I don't see anything in that roadmap. When will it be populated?

More importantly, when will the next release be made? We're waiting on a version that includes PR #1666.

Thanks!

**Comment by jameskairu** [comment]
great

**Comment by zhaoccqq** [comment]
#

**Comment by zhaoccqq** [comment]
> 您好 GCM 社区！👋
> 
> 我们很高兴地宣布，我们已开始维护规划中的 GCM 功能的[路线图](https://github.com/git-ecosystem/git-credential-manager/milestones)。💡**专业提示：**查看路线图时，从右上角的 **Sort** 菜单中选择 **Farthest due date（最远的截止日期**），以按升序查看里程碑。
> 
> 我们希望这将有助于向大家澄清项目的计划工作及其时间表。有几点需要注意：
> 
> 1. 为了与 Git 项目保持一致，我们创建了与 Git 即将发布的版本相吻合的里程碑。我们的目标是持续发布接下来两个版本的具体计划，每次发布完成后都会创建一个新的里程碑。
> 2. 我们正在跟踪但尚未分配给发行版的项目将属于 **Future** 里程碑。
> 3. 此路线图旨在跟踪重大项目。虽然他们没有在路线图上被跟踪，但项目维护者将继续在每周的审查会议上解决/分配时间处理错误和问题。
> 
> 非常感谢您推荐这种方法！
> 
> 问题？在下面评论！我们都在倾听👂。

听了军服

**Comment by cashpilotthrive-hue** [comment]
Validate me

**Comment by sujonislampobasi-coder** [comment]
tk666 লগইন পাসওয়ার্ড ভুলে গেছি
